src/CodeFormatter/Program.cs

```diff
@@ -89,7 +73,7 @@ private static void ListRulesAndAnalyzers(bool listAnalyzers, bool listRules)
 
             if (listAnalyzers)
             {
-                ImmutableArray<DiagnosticDescriptor> diagnosticDescriptors = FormattingEngine.GetSupportedDiagnostics(s_defaultCompositionAssemblies);
+                ImmutableArray<DiagnosticDescriptor> diagnosticDescriptors = FormattingEngine.GetSupportedDiagnostics(OptionsHelper.s_defaultCompositionAssemblies);
```

Comment:
Although private/protected static fields are named with `s_`, I don't think public ones should be. `OptionsHelper.DefaultCompositionAssemblies`?